style.css

```diff
@@ -180,7 +180,293 @@ body {
+}
+
+.work-item-actions {
+  margin-top: 2rem;
+}
+
+.vertical-divider {
+  width: 2px;
+  height: 100%;
+  display: flex;
+  justify-content: space-around;
+}
+
+.skills {
+  display: flex;
+}
+
+.button {
+  height: 40px;
+  width: 140px;
+  margin-top: auto;
+  border: none;
+  color: white;
+  transition: all 0.2s;
+}
+
+.button-primary {
+  background: #7d73fc;
+}
+
+.button:hover {
+  box-shadow: 0 8px 16px 0 #7d73fc7a;
+}
+
+.button:active {
+  background: #5e42d7;
+}
+
+.button:disabled {
+  background: #2a2b33;
+}
+
+.contact {
+  margin: 2rem auto;
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+  width: 60%;
+}
+
+.contact-title {
+  color: #9390ff;
+  font-size: 24px;
+}
+
+.contact-title.desktop {
+  display: block;
+}
+
+.contact-title.mobile {
+  display: none;
+}
+
+.contact-desription {
+  text-align: center;
+  margin: 10px 0;
+}
+
+.contact-form {
+  display: grid;
+  grid-template-columns: 1fr;
+  row-gap: 1rem;
+  width: 100%;
+}
+
+.project-section {
+  font-size: 20px;
+  font-weight: 400;
+  line-height: 28px;
+  letter-spacing: 0;
+  text-align: left;
+}
+
+.skillset {
+  display: flex;
+
+  /* align-items: stretch; Default */
+
+  /* justify-content: space-between; */
+  width: 100%;
+  margin: 0;
+  padding: 0;
+}
+
+.skillset li:last-of-type {
+  border-right: none;
+}
+
+h5 {
+  color: #9390ff;
+  font-family: Roboto, Arial, sans-serif;
+  font-size: 30px;
+  font-weight: 700;
+  line-height: 24px;
+  letter-spacing: 0;
+  text-align: center;
+}
+
+.about-me {
+  font-family: Roboto, Arial, sans-serif;
+  font-size: 20px;
+  font-weight: 400;
+  line-height: 28px;
+  letter-spacing: 0;
+  text-align: center;
+}
+
+#names {
+  display: grid; /* 2. display flex to the rescue */
+
+  /* margin-right: auto; */
+  grid-template-columns: repeat(2, 1fr);
+  column-gap: 1rem;
+  width: 100%;
+}
+
+#fullname {
+  display: none;
+}
+
+.send-button {
+  height: 40px;
+  width: 105px;
+  margin-top: auto;
+  border: none;
+  background: #9390ff;
+  color: white;
+}
+
+.second-footer ul {
+  display: flex;
+  justify-content: center;
+  flex-direction: row;
+  list-style: none;
+  column-gap: 1rem;
+  color: white;
+}
+
+.modal {
+  z-index: 3;
+  display: none;
+  padding-top: 50px;
+  position: fixed;
+  left: 0;
+  top: 0;
+  width: 100%;
+  height: 100%;
+  overflow: auto;
+  color: black;
+  background-color: rgb(0, 0, 0);
+  background-color: rgba(0, 0, 0, 0.4);
+}
+
+.modal-content {
+  margin: auto;
+  background-color: #9390ff;
+  position: relative;
+  padding: 0;
+  outline: 0;
+  width: 800px;
+}
+
+.modal-container {
+  max-height: 80vh;
+  overflow-y: scroll;
+  padding: 20px 20px;
+}
+
+@media only screen and (max-width: 770px) {
+  .nav-bar {
+    display: none;
+  }
+
+  .hamburger-btn {
+    color: white;
+    display: block;
+    font-size: 2em;
+  }
+
+  .technologies {
+    justify-content: flex-start;
+  }
+
+  .page-break {
+    flex-direction: column;
+    align-items: center;
+  }
+
```

Comment:
- [ ] In the mobile version, kindly remove the horizontal scroll present.
![Capture](https://user-images.githubusercontent.com/85095702/165326092-e8f09d51-d53d-4304-8eee-5e6a890b97a5.PNG)

I think it's the background image causing the width to overflow

Reply:
@phelian23 
Could you help me identify the exact place that is causing the overflow please if you don't mind? I don't seem to see the error from my end.